Utilise FTL's native config upgrade functionality (#1683)
pi-hole/docker-pi-hole: README.md

    volumes:
      # For persisting Pi-hole's databases and common configuration file
      - './etc-pihole:/etc/pihole'

>       # Uncomment the below if you have custom dnsmasq config files that you want to persist. Not needed for most starting fresh with Pi-hole v6. If you're upgrading from v5 you and used it before your should keep it enabled at the first v6 container start for a complete migration. Can be turned off afterwards.

Author: ```suggestion
      # Uncomment the below if you have custom dnsmasq config files that you want to persist. Not needed for most starting fresh with Pi-hole v6. If you're upgrading from v5 you and have used this directory before, you should keep it enabled for the first v6 container start to allow for a complete migration. It can be removed afterwards
```
